MyOwnORM/MyOwnORM/Attributes/CustomPrimaryKeyAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public sealed class CustomPrimaryKeyAttribute : Attribute
+    {
+        public string Id { get; }
```

Comment:
I think this functionality should be separated to another attribute like 'Column' where you can specify attribute's name in db for each property(not only for PK)

Reply:
I created 'ColumnAttribute' and 'CustomPrimaryKeyAttribute' because 'CustomPrimaryKeyAttribute' mean constraint and i checked if property has this attribute it should be primary key in db